lib/Jsrt/JsrtDebugManager.cpp

```diff
@@ -154,6 +154,21 @@ HRESULT JsrtDebugManager::DbgRegisterFunction(Js::ScriptContext* scriptContext,
         {
             utf8SourceInfo->SetDebugDocument(debugDocument);
         }
+
+        if (this->debugEventCallback != nullptr && !scriptContext->GetDebugContext()->GetIsReparsingSource())
+        {
+            JsrtDebugEventObject debugEventObject(scriptContext);
+            Js::DynamicObject* eventDataObject = debugEventObject.GetEventDataObject();
+            JsrtDebugUtils::AddSourceMetadataToObject(eventDataObject, utf8SourceInfo);
```

Comment:
This is the case here, but not elsewhere in the file.  That's why I kept is separate.